common/autoware_overlay_rviz_plugin/autoware_minimap_overlay_rviz_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+cmake_minimum_required(VERSION 3.8)
+project(autoware_minimap_overlay_rviz_plugin)
+
+set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};/usr/share/cmake/geographiclib")
+
+find_package(autoware_cmake REQUIRED)
+autoware_package()
+
+find_package(GeographicLib REQUIRED)
+find_package(Qt5 COMPONENTS Network REQUIRED)
+find_package(OGRE REQUIRED)
+
+set(
+    headers_to_moc
+)
+
+set(
+  headers_to_not_moc
+  src/include/minimap.hpp
+  src/include/goal_pose.hpp
+  src/include/tile.hpp
+  src/include/tile_field.hpp
+  src/include/tile_provider.hpp
+  src/include/overlay_utils.hpp
+  src/include/path_overlay.hpp
+)
+
+foreach(header "${headers_to_moc}")
+    qt5_wrap_cpp(display_moc_files "${header}")
+endforeach()
+
+set(
+    display_source_files
+    src/overlay_utils.cpp
+    src/minimap.cpp
+    src/tile.cpp
+    src/tile_field.cpp
+    src/tile_provider.cpp
+    src/goal_pose.cpp
+    src/path_overlay.cpp
+)
+
+ament_auto_add_library(
+    ${PROJECT_NAME} SHARED
+    ${display_moc_files}
+    ${headers_to_not_moc}
+    ${display_source_files}
+)
+
+target_compile_options(
+  ${PROJECT_NAME}
+  PUBLIC
+  "-DVEHICLEMAP_VERSION=\"${vehicle_map_VERSION}\""
```

Comment:
What is this variable for?

Reply:
@mitsudome-r  Probably unnecessary, it came with the template i used as a starter to create the plugin

Reply:
Let's remove it then.